### DISCUSSION TITLE
Possibility to Github Enterprise

### DISCUSSION THREAD

**Discussion by xpdable** [anuraghazra/github-readme-stats#1845] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
We use github enterprise, is this working in ghe? or more config/dev?

**Comment by sban2009** [comment]
I would love to have this as part of enterprise. @anuraghazra

**Comment by rickstaa** [comment, marked as answer]
@xpdable and @sban2009 Thanks for your question. Currently, we do not include stats of organization repositories in the cards. However, this is one of the most requested features (see https://github.com/anuraghazra/github-readme-stats/issues/1). This feature has been implemented in https://github.com/anuraghazra/github-readme-stats/pull/1122, but some things must be fixed. However, you can already use this feature on your own fork by deploying your own Vercel instance (see https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance). You can follow the steps explained in https://github.com/anuraghazra/github-readme-stats/issues/1#issuecomment-981419174 to do this.

**Reply by sban2009** [reply]
@rickstaa will this work on our work accounts in organization domains like [IBM GitHub Enterprise](https://github.ibm.com/)?

**Reply by rickstaa** [reply]
@sban2009 I am not very familiar with Enterprise accounts. You can however try it out using the [graphql explorer](https://docs.github.com/en/graphql/overview/explorer) using the following syntax:

```graphql
query userInfo($login: String!) {
  user(login: $login) {
    name
    login
    contributionsCollection {
      totalCommitContributions
      restrictedContributionsCount
    }
    repositoriesContributedTo(
      first: 1
      contributionTypes: [COMMIT, ISSUE, PULL_REQUEST, REPOSITORY]
    ) {
      totalCount
    }
    pullRequests(first: 1) {
      totalCount
    }
    openIssues: issues(states: OPEN) {
      totalCount
    }
    closedIssues: issues(states: CLOSED) {
      totalCount
    }
    followers {
      totalCount
    }
    repositories(
      first: 100
      ownerAffiliations: [OWNER, COLLABORATOR]
      orderBy: {direction: DESC, field: STARGAZERS}
    ) {
      totalCount
      nodes {
        stargazers {
          totalCount
        }
      }
    }
  }
}
```

```graphql
{"login": "sban2009"}
```

#1122 uses https://docs.github.com/en/graphql/reference/enums#repositoryaffiliation to fetch data from organizations, so if the repositories of your enterprise accounts fall under the `COLLABORATOR` value, the PR should work. You can add your support for getting it merged to #1222. If it does not work, you can create a feature request or a PR to make it work with the Graphql API (see https://docs.github.com/en/enterprise-server@3.3/graphql).

**Reply by rickstaa** [reply]
@sban2009 Did that answer your question?
